src/Between.php

```diff
@@ -164,6 +164,10 @@ public function isValid($value)
     {
         $this->setValue($value);
 
+        if (is_null($this->getMin()) || is_null($this->getMax())) {
```

Comment:
It's better to perform this kind of checks here and whenever like `$smth !== null`